test/include/userobjects/GhostUserObject.h

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef GHOSTUSEROBJECT_H
+#define GHOSTUSEROBJECT_H
+
+#include "GeneralUserObject.h"
+
+//Forward Declarations
+class GhostUserObject;
+
+template<>
+InputParameters validParams<GhostUserObject>();
+
+/**
+ * User object to show Ghosted User Objects on ALL processors or a single processor.
```

Comment:
Not sure I follow this comment.  Do you mean "show ghosted elements" or "show ghosted geometric entities"?